Source/Stationeers.Addons.PluginCompiler/Stationeers.Addons.PluginCompiler.csproj

```diff
@@ -21,7 +21,7 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\Build\PluginCompiler\Debug\</OutputPath>
```

Comment:
Need to undo overwrite

Reply:
Resolved: https://github.com/Erdroy/Stationeers.Addons/pull/3/commits/6a7422ba90e21c6ff3d86ba717c5b3e28314fb3b